wikiclue/src/routes/daily/+page.svelte

```diff
@@ -0,0 +1,182 @@
+<script lang="ts">
+	import HeaderBar from '../../components/headerBar.svelte';
+	import { writable } from 'svelte/store';
+	import { onMount } from 'svelte';
+	import { browser } from '$app/environment';
+	import DailyEnd from '../../components/dailyEnd.svelte';
+
+	const isOverlayOpen = writable(false);
+	let searchTerm = '';
+	const wordsToFind = ['Sample', 'Words']; // this will need to change to actual algo code
+	let incorrectAnswer = false;
+	let gameOver = false;
+	let guessesRemaining = 6;
+	let today = new Date();
+	let maxStreak = 0;
+	let currentStreak = 0;
+	let totalWins = 0;
+	let totalTimesPlayed = 0;
+	let guessDistribution: number[] = [];
+	let statistics = { Played: 0, 'Win %': '0%', 'Current Streak': 0, 'Max Streak': 0 };
+	let monthNames = [
+		'January',
+		'February',
+		'March',
+		'April',
+		'May',
+		'June',
+		'July',
+		'August',
+		'September',
+		'October',
+		'November',
+		'December'
+	];
+
+	onMount(() => {
+		loadDailyWords();
+		getUserInfo();
+	});
+
+	function loadDailyWords() {
```

Comment:
Don't really need to use session storage here. John did that for daily to avoid refresh issues and stuff. For the daily, I think just reading firebase would be fine

Reply:
Fixed